src/tests/ie_test_utils/common_test_utils/ngraph_test_utils.hpp

```diff
@@ -43,6 +43,8 @@ class TransformationTestsF : public  CommonTestUtils::TestsCommon {
     void enable_soft_names_comparison();
 
     std::shared_ptr<ov::Model> function, function_ref;
+    // Aliases to function and function_ref pointers to be more corresponding with ov namespace.
+    std::shared_ptr<ov::Model>&model, &model_ref;
```

Comment:
I'm not sure if this is a good idea, it looks confusing
I think it's better to rename function and function_ref to model, model_ref in the next PR.

Reply:
Added as eye decomposition not use the legacy naming. It can be removed when other composition will be refactored to use `ov` namespace.
Change `function`, `functions_ref `can be big PR and  I don't know when it happens. Now you can use model or functions depends if old or new naming is used.

Reply:
I would recommend to create a jira ticket

Reply:
JIRA story created.